code/game/objects/items/weapons/trays.dm

```diff
@@ -214,9 +214,9 @@
 	return 1
 
 /obj/item/weapon/tray/AltClick(var/mob/user)
+	if (!use_check(user, FALSE)) return

```

Comment:
shit I fucked up. This should be `if (!use_check(user, show_messages = FALSE)) return`